### PR TITLE
Bleach

### PR DESCRIPTION
**Pull request by sanket305**
What does your Instant Answer do?

Provides all the information about  bleach anime

Why did you choose this data source?

It provides best sites about bleach anime

Are there any other alternative (better) data sources?

Yes, But this is the official one

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)

manga and anime lover

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?

No

Are you having any problems? Do you need our help with anything?

NO

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)
screenshot 3

### CODE REVIEW

File: share/goodie/cheat_sheets/json/bleach.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+   "id":"anime",
```
Comment (MariagraziaAlastra) [inline share/goodie/cheat_sheets/json/bleach.json]: @sanket305 Could you change the id to "bleach", perhaps? We already have an IA with this id: https://duck.co/ia/view/anime

File: share/goodie/cheat_sheets/naruto.JSON [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/**
```
Comment (MariagraziaAlastra) [inline share/goodie/cheat_sheets/naruto.JSON]: @sanket305 Can you please remove this file? If you want to make a Naruto Cheat Sheet too it'd be better to do that in a different PR :)

**Comment by MariagraziaAlastra** [comment]
Hey @sanket305, thanks for your contribution!
I just have a few comments.
Also, did you know that now it's not necessary to fill the PR template anymore? Just create your IA Page [here](https://duck.co/ia/new_ia) (you can use the link to your PR as well) and paste the link to it in your first comment here.
You can find more info [in the docs](http://docs.duckduckhack.com/submitting/pull-request.html#submitting-a-pull-request).
Thanks!

File: share/goodie/cheat_sheets/json/bleach.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{
+   "id":"bleach",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/bleach.json]: @sanket305 lets change this to "bleach_cheat_sheet"

File: share/goodie/cheat_sheets/json/bleach.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+       "bleach anime and manga",
+      "bleach anime&manga",
+      "bleach anime to watch"
+   ],
+   "template_type":"reference",
+   "section_order":[
+      "About",
+      "TO read manga",
+      "TO watch anime"
+   ],
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/bleach.json]: @sanket305 lets change these to "To Watch Manga" , "To Watch Anime"

**Comment by moollaza** [comment]
@sanket305 Thanks for contributing and welcome to DuckDuckHack!

I think the scope of this cheat sheet is a little too narrow currently. I imagine queries like this would be better served by a Spice that shows where users can watch various TV shows.

We currently have a GoWatchIt IA that handles movie and TV streaming which doesn't seem to have results for this. Are there any API's that could do somthing similar to GoWatchIt for Mange?

**Comment by moollaza** [comment]
For now I'm closing this because I think there's a better, more useful IA that could be built to better serve this portion of the query space

**Comment by moollaza** [comment]
@sanket305 thanks again for working on this Bleach Cheat Sheet! I'm sorry we couldn't merge it, but you did a great job. We at DuckDuckGo, and the DuckDuckHack community really appreciate you taking the time to contribute.

We could really use your help on something else now. We want to increase our Instant Answer coverage for as many programming languages and platforms as possible. We want an Objective-C Cheat Sheet and I think you'd similarly do a great job with building that. Please update the [Programming IA Coverage table](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage) (mark it with a 'd') if you'd be willing to take it up!

I think it is a good fit for you, but I'm of course not entirely sure, so I'd love to chat if you're around this week or next. Please message me on slack (@moollaza) or send me an email (moollaza@duckduckgo.com) if you'd be interested in talking. 

Thanks again!
